app/services/notifications/update.rb

```diff
@@ -24,7 +24,9 @@ def call
       new_json_data = notifications.first.json_data || {}
       new_json_data[notifiable.class.name.downcase] = public_send("#{notifiable.class.name.downcase}_data", notifiable)
       new_json_data[:user] = user_data(notifiable.user)
-      new_json_data[:organization] = organization_data(notifiable.organization) if notifiable.is_a?(Article) && notifiable.organization_id
+      add_organization_data = notifiable.is_a?(Article) && notifiable.organization_id
```

Comment:
Should we make this more resilient and check for the actual organization in addition to the organization_id? We do this a couple of spots in the code for other models and I have been fixing them when the related model gets deleted and the code breaks bc it is expecting it due to the presence of the id.

Reply:
I'm not familiar with this error or this code (I just noticed the line was too long), can you point me to the error or another example in the code? I can add the check here as well

Reply:
Here is a scenario I had to fix a couple of weeks ago https://github.com/thepracticaldev/dev.to/pull/5264

Reply:
Ok, i'll address it in a separate PR about `Notifications::Update` because I've already noticed a couple more optimizations that can be applied